lib/msal-browser/test/mochaSetup.js

```diff
@@ -2,4 +2,6 @@ require("@babel/register")({ extensions: ['.js', '.jsx', '.ts', '.tsx'] });
 require("jsdom-global")("", {
     url: "https://localhost:8081/index.html"
 });
+window.crypto = require("@trust/webcrypto");
+window["msCrypto"] = require("@trust/webcrypto");
```

Comment:
I'm not sure you would actually want to do this for `msCrypto`? Since the APIs are different?

Reply:
Is there a package I can use for this instead? I was really just going to check for existence of the API using this.

Reply:
Removed for now - I will add mocks and more later.